packages/booking-dispatcher/README.md

```diff
@@ -0,0 +1,11 @@
+# `booking-dispatcher`
+
+> TODO: description
+
+## Usage
+
+```
+const bookingDispatcher = require('booking-dispatcher');
+
+// TODO: DEMONSTRATE API
```

Comment:
Start the package together with engine-server, engine and engine-ui. This will use the data from the exel file and present the real data from last year in Ljusdal. 
Why? To see and work with real data. I will update so each day we get new data from that day a year ago :)

Reply:
Yes, ok! I'm just having some problems understanding what value it brings to just see the bookings pop up on the screen...? Also, maybe you can put what you wrote into the readme, together with the environment variables instructions, etc. :)